### DISCUSSION TITLE
Change the color of the menu and symbol bar ...

### DISCUSSION THREAD

**Discussion by boa2145** [geany/geany#3529] [General]
Hi, Is it possible to change the color of the menu and symbol bar at the top of the IDE? I use the dark scheme on Windows 11 and would like to use it for the menu and symbol bar of the app "Geany", too, but I don't know how it works. Regards ... Gregor

**Comment by elextr** [comment]
The theme for all elements except the edit pane is set by the GTK theme, see [the new FAQ](https://github.com/geany/www.geany.org/blob/8f469954d7d1ee3c288c9bbf2c2d16368d724b91/page_content/documentation/faq.md) for an as yet not merged (ping @eht16) FAQ for how to do it.

**Comment by boa2145** [comment]
Hi Elextr, Thanks for your reply.

**Reply by ghost** [reply]
Hello, I followed the FAQ guideline **How to change the GTK theme?** on Windows 10. 
To make it work on my machine, I had to change the location of file `settings.ini` from `c:\Users/<YOUR_USERNAME>\AppData\Local\gtk-3.0\` to `c:\Users/<YOUR_USERNAME>\AppData\Roaming\gtk-3.0\` . I would assume that this change is also required to Windows 11.

**Comment by eht16** [comment]
@elextr thanks for the reminder. I merged and deployed the PR and it is now available at https://www.geany.org/documentation/faq/#how-to-change-the-gtk-theme.
